api/tests/routes/backoffice/pro_users_test.py

```diff
@@ -320,6 +320,20 @@ def test_no_user_offerers(self, authenticated_client, pro_user):
             response = authenticated_client.get(url)
         assert response.status_code == 200
 
+    def test_get_user_offerers_in_different_offerer_status(self, authenticated_client, pro_user):
+        offerers_factories.UserOffererFactory(user=pro_user, offerer=offerers_factories.PendingOffererFactory())
+        url = url_for(self.endpoint, user_id=pro_user.id)
+
+        with assert_no_duplicated_queries():
+            response = authenticated_client.get(url)
+
+        assert response.status_code == 200
+
+        rows = html_parser.extract_table_rows(response.data, parent_class="user_offerers-tab-pane")
+        assert len(rows) == 2
+
+        assert rows[0]["Statut structure"] != rows[1]["Statut structure"]
```

Comment:
En vérifiant exactement le contenu, cela permet de s'assurer que la construction du badge dans le jinja est bonne :
```suggestion
        assert rows[0]["Statut structure"] == "Validé"
        assert rows[1]["Statut structure"] == "En attente"
```